morango/migrations/0022_rename_instance_fields.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2023-01-31 18:53
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('morango', '0021_store_partition_index_create'),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name='syncsession',
+            old_name='client_instance',
+            new_name='client_instance_json',
+        ),
```

Comment:
May need to make some tweaks in Kolibri for this: https://github.com/learningequality/kolibri/search?q=client_instance (should come up during tests after integration)